apis/apps/v1beta3/emqxbroker_types.go

```diff
@@ -33,46 +33,65 @@ type EmqxBrokerTemplate struct {
 	//+kubebuilder:default:="public"
 	Password string `json:"password,omitempty"`
 
+	// ExtraVolumes for mounting extra volumes like secrets and/or configmaps
 	// See https://github.com/emqx/emqx-operator/pull/72
 	ExtraVolumes []corev1.Volume `json:"extraVolumes,omitempty"`
 	// See https://github.com/emqx/emqx-operator/pull/72
 	ExtraVolumeMounts []corev1.VolumeMount `json:"extraVolumeMounts,omitempty"`
 
+	// EmqxConfig for EMQX cluster configurations
 	EmqxConfig EmqxConfig `json:"config,omitempty"`
-	Args       []string   `json:"args,omitempty"`
+	// Args define arguments for the command
+	Args []string `json:"args,omitempty"`
 
-	SecurityContext *corev1.PodSecurityContext  `json:"securityContext,omitempty"`
-	Resources       corev1.ResourceRequirements `json:"resources,omitempty"`
+	// SecurityContext defines pod-level security attributes and common container settings
+	SecurityContext *corev1.PodSecurityContext `json:"securityContext,omitempty"`
+	// Resources specify cpu and memory resources
+	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
 
+	// ReadinessProbe indicates whether the application running in the container is ready to accept requests
 	ReadinessProbe *corev1.Probe `json:"readinessProbe,omitempty"`
-	LivenessProbe  *corev1.Probe `json:"livenessProbe,omitempty"`
-	StartupProbe   *corev1.Probe `json:"startupProbe,omitempty"`
+	// LivenessProbe indicates if the container is operating
+	LivenessProbe *corev1.Probe `json:"livenessProbe,omitempty"`
+	// StartupProbe indicates whether the application running in the container has started
+	StartupProbe *corev1.Probe `json:"startupProbe,omitempty"`
 
-	ServiceTemplate ServiceTemplate    `json:"serviceTemplate,omitempty"`
-	ACL             []string           `json:"acl,omitempty"`
-	Modules         []EmqxBrokerModule `json:"modules,omitempty"`
+	// ServiceTemplate defines a logical set of Pods and a policy by which to access them
+	ServiceTemplate ServiceTemplate `json:"serviceTemplate,omitempty"`
+	// ACL defines ACL rules
+	ACL []string `json:"acl,omitempty"`
+	// Modules define functional modules for EMQX broker
+	Modules []EmqxBrokerModule `json:"modules,omitempty"`
 }
 
 // EmqxBrokerSpec defines the desired state of EmqxBroker
 type EmqxBrokerSpec struct {
 	//+kubebuilder:default:=3
 	Replicas *int32 `json:"replicas,omitempty"`
 
+	// ImagePullSecrets For pulling EMQX broker image
```

Comment:
```suggestion
	// ImagePullSecrets For pulling image
```
ImagePullSecrets is not only for EMQX image, it can be used for all images